README.md

```diff
@@ -504,6 +505,17 @@ tns run ios --emulator
 
 [Back to Top][1]
 
+Extending the CLI
+===
+
+The NativeScript CLI lets you extend its behavior and customize it to fit your needs by using [hooks](https://en.wikipedia.org/wiki/Hooking).
+
+When you run one of the [extendable commands](#commands-with-hooking-support) (for example, `tns build`), the CLI checks for hooks and executes them. Plugins can also use hooks to control the compilation of the application package.
```

Comment:
The anchor link will not work here. Either remove the linking syntax or change it to point to `https://github.com/NativeScript/nativescript-cli/blob/master/extending-cli.md#commands-with-hooking-support`

Also, I'm wondering whether the link should be from the release branch? (both here and below)